content/blog/2019-02-18-stats19.md

```diff
@@ -0,0 +1,1014 @@
+    radius: 6,
+    color: "#000",
+    weight: 1,
+    opacity: 1,
+    fillOpacity: 0.5
+  };
+  var layer = L.geoJSON(json, {
+    pointToLayer: function (feature, latlng) {
+        return L.circleMarker(latlng, geojsonMarkerOptions);
+    },
+    style: function(feature) {
+      switch (feature.properties.accident_severity) {
+          case "Serious": return {color: "#FEB24C"};
+          case "Fatal":   return {color: "#BD0026"};
+       }
+    }
+  }).addTo(map);
+  map.fitBounds(layer.getBounds());
+  var legend = L.control({position: "bottomright"});
+  legend.onAdd = function (map) {
+    var div = L.DomUtil.create("div", "info legend"), labels = [];
+    labels.push("<h6>accident_severity</h6>");
+    labels.push("<i style='background:#FEB24C'></i>Serious");
+    labels.push("<i style='background:#BD0026'></i>Fatal");
+    div.innerHTML = labels.join("<br>");
+    return div;
+  };
+  legend.addTo(map);
+  var info = L.control();
+  info.onAdd = function (map) {
+    this._div = L.DomUtil.create("div", "info");
+    this.update();
+    return this._div;
+  };
+  info.update = function (props) {
+    this._div.innerHTML = "<h6>Crashes in West Yorkshire (2017)</h6>";
+  };
+  info.addTo(map);
+</script>
+
+<style>
+.info { padding: 6px 8px; font: 14px/16px Arial, Helvetica, sans-serif; background: white; background: rgba(255,255,255,0.8); box-shadow: 0 0 15px rgba(0,0,0,0.2); border-radius: 5px; } .info h4 { margin: 0 0 5px; color: #777; }
+.legend { text-align: left; line-height: 18px; color: #555; } .legend i { width: 18px; height: 18px; float: left; margin-right: 8px; opacity: 0.7; }</style>
+
+</div>
+
+<!--/html_preserve-->
+
+#### Conclusion
+
+[`stats19`](https://github.com/ropensci/stats19) provides access to a
+reliable and official road safety dataset. As covered in this post, it
+helps with data discovery, download, cleaning and formatting, allowing
+you to focus on the real work of analysis (see the package’s
+introductory
+[vignette](https://itsleeds.github.io/stats19/articles/stats19.html) for
+more on this). In our experience, 80% of time spent using STATS19 data
+was spent on data cleaning. Hopefully, now the data is freely available
+in a more useful form, 100% of the time can be spent on analysis\! We
+think it could help many people, especially, including campaigners,
+academics and local authority planners aiming to make the roads of the
+UK and the rest of the world a safe place for all of us.
+
+There are many possible next steps, including:
+
+  - Comparing these datasets with interventions such as 20 mph zones
+    (Grundy et al. 2009) and links with street morphology (Sarkar,
+    Webster, and Kumari 2018).
+  - The creation of more general software for accessing and working with
+    road crash data worldwide.
+  - Making the data even more available by provide the data as part of
+    an interactive web application, a technique successfully used in the
+    Propensity to Cycle Tool (PCT) project hosted at
```

Comment:
Propensity to Cycle Tool (PCT) project hosted at [www.pct.bike/](http://www.pct.bike/)
->
`[Propensity to Cycle Tool](https://www.pct.bike/) project`

Reply:
Done.